README.md

```diff
@@ -1,21 +1,60 @@
 # erouska-android
 
-Erouška for Android
+Erouška ("rouška" means "face mask" in Czech) is an app which helps to fight against COVID-19. When turned on, it scans other users of the app via Bluetooth and saves this data locally. When some user tests positive for COVID-19, he/she is asked to upload their data. Then operators from the state can see phone numbers of users of the app which were in close contact with the infected and can call them about further measures.
```

Comment:
Možná bych state nahradil za něco jako healthcare authorities. Bude to min big Brother 😁